appveyor.yml

```diff
@@ -0,0 +1,24 @@
+branches:
+  only:
+  - master
+
+environment:
+  nodejs_version: '5.7.1'
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - set CI=true
+  - set PATH=%APPDATA%\npm;%PATH%
+  - npm install
+
+build: off
+version: '{build}'
+shallow_clone: true
+clone_depth: 1
+
+test_script:
+  - node --version
+  - npm --version
+  - npm run lint -s
+  - npm run test -s
+#  - npm run test:func -s
```

Comment:
windows does not have `cat` command so there is no sense to test against it's output =)